code/game/gamemodes/cult/cult.dm

```diff
@@ -26,6 +26,8 @@ var/global/list/all_cults = list()
 		var/mob/living/simple_animal/hostile/guardian/G = mind.current
 		if(!iscultist(G.summoner))
 			return 0 //can't convert it unless the owner is converted
+	if(isgolem(mind.current))
+		return 0

```

Comment:
return FALSE. But then it would be better to refactor the whole proc if you change this

Reply:
Thanks.